composer.json

```diff
@@ -15,6 +15,12 @@
             "homepage": "https://github.com/jackalope/jackalope-jackrabbit/contributors"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url":  "https://github.com/richard-fizz/jackalope-doctrine-dbal.git"
```

Comment:
you need to do this with the phpcr-api-tests and the phpcr-utils instead of jackalope-doctrine-dbal. and change the version requirement on phpcr-api-tests to dev-master to get your changes in. same with phpcr-utils. (if you did the changes in a branch, it would be dev-<that-branch-name>)